packages/logs/src/domain/logsCollection/console/consoleCollection.spec.ts

```diff
@@ -58,7 +58,6 @@ describe('console collection', () => {
     console.error('foo', 'bar')
 
     expect(rawLogsEvents[0].rawLogsEvent.error).toEqual({
-      origin: ErrorSource.CONSOLE,
```

Comment:
💭 Maybe we could have one test checking on `rawLogsEvents[0].rawLogsEvent` to ensure `origin` is `ErrorSource.CONSOLE` when using `console.error` ?

Reply:
I'd say that it is addressed by the test above: https://github.com/DataDog/browser-sdk/blob/991e97418579256bf36d38e3cbdaf77c7d6052c1/packages/logs/src/domain/logsCollection/console/consoleCollection.spec.ts#L31-L46

However, we could have this test for each status and not only for log.
wdyt?

Reply:
Yes I was thinking that adding `console.error` would be good, but indeed `console.*` is even better.